File: TEST data for large repo/repo-Copy5/Blueprism/BluePrism.Api/BluePrism.Api.Models/ResourceUtilizationParameters.cs
namespace BluePrism.Api.Models
{
    using System;
    using System.Collections.Generic;

    public class ResourceUtilizationParameters
    {
        public DateTime StartDate { get; set; }
        public IReadOnlyCollection<Guid> ResourceIds { get; set; }
        public int? PageNumber { get; set; }
        public int? PageSize { get; set; }
    }
}

File: TEST data for large repo/REPO_Copy-6/Blueprism/BluePrism.Server.Domain.Models/ResourceUtilizationParameters.vb
Imports System.Runtime.Serialization

<Serializable()>
<DataContract([Namespace]:="bp")>
Public Class ResourceUtilizationParameters
    <IgnoreDataMember>
    <CLSCompliantAttribute(False)>
    Public Property StartDate As DateTime
    <IgnoreDataMember>
    <CLSCompliantAttribute(False)>
    Public Property ResourceIds As IEnumerable(Of Guid)
    <IgnoreDataMember>
    <CLSCompliantAttribute(False)>
    Public Property PageNumber As Integer?
    <IgnoreDataMember>
    <CLSCompliantAttribute(False)>
    Public Property PageSize As Integer?
End Class
